syntax = "proto3";
import "k2tree.proto";
package proto_msg;


option java_package = "com.github.cristobalm.RDFEWK2C.proto";
option java_outer_classname = "PredicatesIndexCacheMsg";


message PredicatesIndexCache{
    repeated K2Tree k2tree = 1;
};